deployment/pom.xml

```diff
@@ -34,7 +34,7 @@
 
         <dependency>
             <groupId>io.quarkus</groupId>
-            <artifactId>quarkus-smallrye-openapi-spi</artifactId>
+            <artifactId>quarkus-smallrye-openapi-deployment</artifactId>
```

Comment:
Why deployment? `@OpenApiFilter` is in the runtime library.
```suggestion
            <artifactId>quarkus-smallrye-openapi</artifactId>
```

Reply:
Deployment modules usually depend on other extension's deployment modules